frontend/src/modules/history/components/Translation.js

```diff
@@ -142,31 +142,52 @@ export class TranslationBase extends React.Component<InternalProps, State> {
             rel='noopener noreferrer'
             onClick={ (e: SyntheticMouseEvent<>) => e.stopPropagation() }
         >
-            <img
-                src={ translation.user_gravatar_url_small }
-                alt=''
-                height='44'
-                width='44'
-            />
+            { translation.user }
         </a>
     }
 
-    renderUser() {
-        const { translation } = this.props;
+    toggleComments = (event: SyntheticMouseEvent<>) => {
+        event.stopPropagation();
+        this.setState((state) => {
+            return { isCommentVisible: !state.isCommentVisible };
+        });
+    }
 
-        if (!translation.uid) {
-            return <span>{ translation.user }</span>;
+    renderCommentButton(commentCount: number) {
```

Comment:
`renderCommentToggle` would be consistent with the naming used by the Diff toggle.